Makefile

```
@@ -3,14 +3,28 @@
 # Ensure `cd` works properly by forcing everything to be executed in a single shell.
 .ONESHELL:
 
-# Build KaTeX.
-all:
+all: src/KaTeX src/workbox src/icon-192.png src/icon-512.png
+
+# Vendor KaTeX dependencies.
+src/KaTeX:
 	set -e
 	curl -L -O "https://github.com/KaTeX/KaTeX/releases/download/v0.16.4/katex.zip"
 	unzip katex.zip
 	rm katex.zip
 	mv katex src/KaTeX
 
+# Vendor workbox dependencies.
+src/workbox: src/workbox/workbox-window.prod.mjs src/workbox/workbox-sw.js src/workbox/workbox-core.prod.js src/workbox/workbox-routing.prod.js src/workbox/workbox-strategies.prod.js
```

```suggestion
src/workbox: src/workbox/workbox-window.prod.mjs src/workbox/workbox-window.prod.mjs.map src/workbox/workbox-sw.js src/workbox/workbox-core.prod.js src/workbox/workbox-routing.prod.js src/workbox/workbox-strategies.prod.js
```
I also needed to add this file to avoid a 404 error.

---

Yes, I left out the source maps because that's just a warning from the browser DevTools; however they could come in handy while debugging in production.

I'm adding the maps of all workbox dependencies (the service worker is also raising the same warning for each one.)

---

Ah, I see, thanks.